src/components/Utils/orderResource.js

```diff
@@ -17,26 +15,33 @@ const saveOrder = (order, mutator) => {
   return method(order);
 };
 
+export const getFullOrderNumber = ({ poNumberPrefix = '', poNumberSuffix = '', poNumber = '' }) => (
```

Comment:
it will throw an exception if null or undefined is passed. Maybe it makes sense to add a default value
```suggestion
export const getFullOrderNumber = ({ poNumberPrefix = '', poNumberSuffix = '', poNumber = '' } = {}) => (
```

Reply:
correct, missed it